### ISSUE TITLE
Headings - should render italics (and not bold)

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#400]
**Is your feature request related to a problem? Please describe.**
The headings will currently render italics and bold. We only want to be able to render italics.

**Describe the solution you'd like**
- The italics should use the `i` element
- We should use the italic subset of the font being used. https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_It.woff2, https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_It.woff or https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_It.woff2, https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_It.woff
- Bold markdown should not be rendered into bold

**Comment by BogdanDogaru** [comment]
Blocked pending a wider UX&D discussion on how bold/italics will be handled.

**Comment by BogdanDogaru** [comment]
~Not blocked anymore: Headlines and headings should not have the ability to display bold text - only italics.~ see below

**Comment by jtart** [comment]
@BogdanDogaru it is blocked by Fragments only being able to render bold and italics. Need to be able to pass an override.

Need to create an issue for this.

**Comment by bcmn** [comment]
Closing in favour of #1276. Will try & prioritise refining this & get an approach on the issue mentioned by Jordan above defined.
